File: subscriber.js
// subber.js
var zmq = require('zeromq')
  , sock = zmq.socket('sub');

sock.connect('tcp://127.0.0.1:3000');
sock.subscribe('Bouncy Ball');
console.log('Subscriber connected to port 3000');

const gameDraw = require('./gamefiles/gamedraw');


sock.on('message', function(topic, message) {
  //console.log('received a message related to:', String(topic), 'containing message:', JSON.parse(String(message)));
    gameDraw(JSON.parse(String(message)));
});

File: subscriber.py
import zmq
import time
from gamefiles.gamedraw import gamedraw
import json
ctx = zmq.Context()
sock = ctx.socket(zmq.SUB)


sock.connect("tcp://127.0.0.1:3000")
sock.subscribe("Bouncy Ball") # Subscribe to all topics

while True:
    data = sock.recv_string()
    if(data != "Bouncy Ball"):
        gamedraw(json.loads(data))
    #gamedraw(json.loads(data[1]))

sock.close()
ctx.term()